### PR TITLE
class names are case sensitive (BC break)

### PR DESCRIPTION
**Pull request by dg**

### CODE REVIEW

File: src/DI/ContainerBuilder.php [hunk 1]
```diff
@@ -445,6 +445,18 @@ private function resolveEntityClass($entity, $recursive = array())
 	}
 
 
+	private function checkCase($class)
```
Comment (fprochazka) [inline src/DI/ContainerBuilder.php]: so, this throws an error, when I supply wrongly cased classname in config or into service definition?

Comment (dg) [inline src/DI/ContainerBuilder.php]: or into ContainerBuilder::getByType()

Comment (fprochazka) [inline src/DI/ContainerBuilder.php]: Great :+1:

Comment (dg) [inline src/DI/ContainerBuilder.php]: Uff :-)

**Comment by JanTvrdik** [comment]
:+1: For doing the check in compile-time only.

**Comment by dg** [comment]
Check is done in compile-time only.

**Comment by JanTvrdik** [comment]
I know, that's why I gave you :+1: I didn't realize that the sentence has other meaning as well :smile:

**Comment by fprochazka** [comment]
I just hope I don't have any typos in my extensions :D better check them :)

**Pull request merged**